frontend/src/pages/DataCatalog/dataset_explorer/DataSourceListing.module.scss

```diff
@@ -24,6 +24,7 @@
 
 .DatasetTitle {
   font-size: 20px;
+  font-family: "Taviraj", serif !important;
```

Comment:
Question not specific to this PR: is it possible to define this font globally somewhere and reference that? iirc we use this font all over the place

Reply:
Yeah I'm actually kinda confused about the font scheme throughout the static pages... I'll file a FE backlog issue

Reply:
https://github.com/SatcherInstitute/health-equity-tracker/issues/567